ui/src/components/PostModal.tsx

```diff
@@ -140,13 +140,13 @@ const ReportButton: React.FC<{ post: Post }> = ({
     const auth = useAuth();
     const reportPostMutation = useReportPostMutation();
 
-    const onClick = (e: any) => {
+    const onClick = (e: { preventDefault(): void }) => {
```

Comment:
This is fine but it might be more concise and expressive to use `MouseEvent`

Reply:
It told me off when I tried to do that (or maybe that was just `Event`) - I'll merge and then fix in a bit